content/20.results.md

```diff
@@ -98,3 +98,16 @@ Outside of the primary range of our analyses, the two names of 2020 PSB keynote
   For each region, the mean predicted probability of Pubmed articles is shown as teal LOESS curve, and the mean probability and 95% confidence interval of the ISCB honoree predictions are shown as dark circles and vertical lines (B).
 
 ](https://raw.githubusercontent.com/greenelab/iscb-diversity/master/figs/region_breakdown.png){#fig:region_breakdown}
+
+
+### Assessing Affiliation Diversity of Authors and Honorees
+We observed an increasing proportion of author affiliation located in East Asia & Pacific (Fig. {@fig:affiliation_breakdown}A, left).
+This was primarily driven by publications in Bioinformatics and BMC Bioinformatics (see [notebook](https://greenelab.github.io/iscb-diversity/15.analyze-affiliation.html#sup_fig_s6)).
+Among ISCB keynote speakers and fellows, we found that the majority of honorees are affiliated with institutions and companies from North America (Fig. {@fig:affiliation_breakdown}A, right).
+Noting the small sample size, we found discrepancies between the group of honoree composition versus PubMed. 
+Compared to Pubmed authors, ISCB keynote speakers and fellows have a higher proportion of individuals affiliated with North American institutions and companies (Fig. {@fig:affiliation_breakdown}B) and lower proportion of individuals affiliated with institutions or companies from Europe & Central Asia and East Asia & Pacific. 
```

Comment:
```suggestion
ISCB keynote speakers and fellows had a higher proportion of individuals affiliated with North American institutions and companies (Fig. {@fig:affiliation_breakdown}B) and lower proportion of individuals affiliated with institutions or companies from Europe & Central Asia and East Asia & Pacific than authors. 
```

Reply:
I left "Compared to authors" at the beginning of this sentence for more direct reading.

Reply:
👍